charts/pxc-db/README.md

```diff
@@ -1,16 +1,149 @@
+| `haproxy.imagePullSecrets`             | HAProxy Container pull secret                                                | `[]`                                      |
+| `haproxy.annotations`             | HAProxy Pod user-defined annotations                                         | `{}` |
+| `haproxy.priorityClassName`       | HAProxy Pod priority Class defined by user                                   |  | 
+| `haproxy.labels`                  | HAProxy Pod user-defined labels                                              | `{}` |
+| `haproxy.readinessDelaySec`       | HAProxy Pod delay for readiness probe in seconds                             | `15` |
+| `haproxy.livenessDelaySec`        | HAProxy Pod delay for liveness probe in seconds                             | `300` |
+| `haproxy.forceUnsafeBootstrap`        | Order HAProxy Pods to override the previous Pod crash                             | `false` |
+| `haproxy.resources`                     | HAProxy Pods resource requests and limits                                    | `{}`                                      |
+| `haproxy.nodeSelector`                  | HAProxy Pods key-value pairs setting for K8S node assingment                 | `{}`                                      |
+| `haproxy.affinity.antiAffinityTopologyKey` | HAProxy Pods simple scheduling restriction on/off for host, zone, region         | `"kubernetes.io/hostname"` |
+| `haproxy.affinity.advanced` | HAProxy Pods advanced scheduling restriction with match expression engine          | `{}` |
+| `haproxy.tolerations`                   | List of node taints to tolerate for HAProxy Pods                       | `[]`                                      |
+| `haproxy.gracePeriod`                   | Allowed time for graceful shutdown                       | `600`                                      |
+| `haproxy.podDisruptionBudget.maxUnavailable` | Instruct Kubernetes about the failed pods allowed quantity           | `1`                                      |
+| `haproxy.persistence.enabled` | Requests a persistent storage (`hostPath` or `storageClass`) from K8S for HAProxy Pods  | `true`                                      |
+| `haproxy.persistence.hostPath` | Sets datadir path on K8S node for all HAProxy Pods. Available only when `pxc.persistence.enabled: true` |                             |
+| `haproxy.persistence.storageClass` | Sets K8S storageClass name for all HAProxy Pods PVC. Available only when `pxc.persistence.enabled: true` | `-`                      |
+| `haproxy.persistence.accessMode` | Sets K8S persistent storage access policy for all HAProxy Pods | `ReadWriteOnce`                      |
+| `haproxy.persistence.size` | Sets K8S persistent storage size for all HAProxy Pods | `8Gi`                      |
+| `haproxy.persistence.size` | Sets K8S persistent storage size for all HAProxy Pods | `8Gi`                      |
```

Comment:
This seems to be listed twice.
Also notice that Mykola says that haproxy should not create pvc's at all - https://jira.percona.com/browse/K8SPXC-400
so probably the behaviour in operator and helm charts will need to be changed.

Reply:
I've removed duplicated entry.
Actually volume settings for haproxy is a result of using the same structure PodSpec for all pod types (haproxy, pxc, proxysql). Maybe will be changed in the future.